packages/tasks_panel/test/view/tasks-tree.spec.ts

```diff
@@ -65,12 +71,16 @@ describe("TasksTree class", () => {
       const taskProvider = new MockTasksProvider(copyTasks);
       const tasksTree = new TasksTree(taskProvider);
       const items = await tasksTree.getChildren();
-      expect(items.length).eq(2);
-      expect(find(items, ["label", copyTasks[0].__intent])).to.be.exist;
-      expect(find(items, ["label", copyTasks[3].__intent])).to.be.exist;
+      expect(items.length).eq(1);
+      expect(items[0] instanceof ProjectTreeItem).to.be.true;
+      expect(items[0].label).to.be.equal(wsFolder);
```

Comment:
from English grammar perspective you would say:
`expect X to equal Y`
not
`expect X to be equal Y`

Note the connecting "chain" words in chai are completely optional are meant only to increase readability
- https://www.chaijs.com/api/bdd/ ("chains" section0